.pipelines/ci/templates/build-powertoys-steps.yml

```diff
@@ -7,6 +7,12 @@ steps:
   submodules: true
   clean: true
 
+- task: UseDotNet@2
+  displayName: 'Use .NET 6 SDK'
```

Comment:
Why are we using .NET 6 and then .NET 8 after?

Reply:
The dotnet tool for the XamlStyler requires .NET 6 to run. Otherwise it all blows up :)